compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -205,8 +206,8 @@ wOptsUnset =
   ]
 
 
-adjustDynFlags :: Options -> FilePath -> DynFlags -> DynFlags
-adjustDynFlags options@Options{..} tmpDir dflags
+adjustDynFlags :: Options -> FilePath -> FilePath -> DynFlags -> DynFlags
```

Comment:
Taking these two file paths positionally without newtyping them makes me a bit nervous.

Reply:
I’ve newtyped the version header.

Reply:
Thank you :)